src/pages/HomePage/AuthForms/Login.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import TextField from "@material-ui/core/TextField";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import Typography from "@material-ui/core/Typography";
+import Container from "@material-ui/core/Container";
+import { useFormik } from 'formik';
+
+const validate = (values: any)=> {
+  const errors = {};
+  if (!values.password) {
+    errors.password = 'Required';
+  } else if (values.password.length > 10) {
+    errors.password = 'Must be 10 characters or less';
+  }
+
+
+  if (!values.email) {
+    errors.email = 'Required';
+  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
+    errors.email = 'Invalid email address';
+  }
+
+  return errors;
+};
+
+const SignIn: React.FC = () => {
+  const formik = useFormik({
+    initialValues: {
+      password: '',
+      email: '',
+    },
+    validate,
+    onSubmit: values => {
+      alert(JSON.stringify(values, null, 2));
+    },
+  });
+
+  return (
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
+      <div>
```

Comment:
Try to avoid using `divs` - if you use it only to wrap other elements, then you can just use `<></>`, else use `Box` from mui